charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -305,6 +305,16 @@ storageClasses: []
 #   parameters:
 #     encrypted: "true"
 
+volumeSnapshotClasses: []
+# Add VolumeSnapshotClass resources like:
+# - name: ebs-vsc
+#   # annotation metadata
+#   annotations:
+#     snapshot.storage.kubernetes.io/is-default-class: "true"
+#   # deletionPolicy must be specified
+#   deletionPolicy: Delete
+#   parameters:
```

Comment:
`parameters` doesn't seem to do anything

Reply:
Updated [volumesnapshotclass.yaml](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1540/files#diff-bc72d4c244db20dcb90c0154f8c95c6a674f59deebd0a84e4366aeec0db8afc7) and testing.